### ISSUE TITLE
Resolve ambiguity with bitmap naming.

### ISSUE THREAD

**Issue by tgruben** [pilosa/pilosa#1144]
### Description
Currently in pilosa we have two different bitmaps, pilosa.Bitmap defined in bitmap.go and pilosa.roaring.Bitmap defined in roaring.go. With some development tools this can be confusing for they do not provide the package level information when presenting the type. I propose a naming change of the roaring bitmap to RoaringBitmap to clear the confusion.
### Success criteria (What criteria will consider this ticket closeable?)
Be able distinguish between the different types without the aid of compiler

**Comment by travisturner** [comment]
No, we can't do this because of stuttering, but we can rename `pilosa.Bitmap`; it seems to actually represent a `Row`, but that should be reviewed.
